rozy/src/app.rs

```diff
@@ -30,12 +30,7 @@ pub struct App {
     executable_path: String,
 }
 
-pub fn find_app(app: &String, version: &Option<String>) -> Result<App> {
-    if version.is_some() {
```

Comment:
If we've made it this long with nobody caring about this going missing I figured we can just drop it.